yb-voyager/cmd/importDataFileTaskImporter.go

```diff
@@ -41,13 +41,17 @@ type FileTaskImporter struct {
 	importBatchArgsProto *tgtdb.ImportBatchArgs
 	workerPool           *pool.Pool
 
+	isTableColocated             bool
+	colocatedImportBatchQueue    chan func()
```

Comment:
can we add a comment here - `colocatedImportBatchQueue is shared across all tasks`